tasks/actorstate/miner/context.go

```diff
@@ -49,23 +50,25 @@ func NewMinerStateExtractionContext(ctx context.Context, a actorstate.ActorInfo,
 	}
 
 	return &MinerStateExtractionContext{
-		PrevState: prevState,
-		PrevTs:    prevTipset,
-		CurrActor: &a.Actor,
-		CurrState: curState,
-		CurrTs:    a.Current,
+		PrevState:     prevState,
+		PrevTs:        prevTipset,
+		CurrActor:     &a.Actor,
+		CurrState:     curState,
+		CurrTs:        a.Current,
+		PreviousState: true,
 	}, nil
 }
 
 type MinerStateExtractionContext struct {
 	PrevState miner.State
 	PrevTs    *types.TipSet
 
-	CurrActor *types.Actor
-	CurrState miner.State
-	CurrTs    *types.TipSet
+	CurrActor     *types.Actor
+	CurrState     miner.State
+	CurrTs        *types.TipSet
+	PreviousState bool
```

Comment:
nit: A slightly more expressive name:
```suggestion
	PreviousStateExists bool
```